### ISSUE TITLE
How to export feature maps of RPN to TensorBoard?

### ISSUE THREAD

**Issue by YutingLin** [smallcorgi/Faster-RCNN_TF#192]
Hi,

I am trying to detect some small objects(such as 20-40 pixels) using Faster-RCNN. I tried to lower the anchor size (anchor_scales) from [8, 16, 32](default) to [2,4,8,16,32]. This adjustment had some improvement, but still not good enough. 
Is the quality of training data that limits the performance, or the size of anchor?

Thus, I want to find out the reason by visualizing the loss and feature maps to TensorBoard. I have exported loss to scalar_summary in TensorBoard, but got stuck in exporting in exporting feature maps using image_summary. 

Does anyone know how to do this?
Thanks.

BTW,
I added some lines for exporting loss and feature maps to TensorBoard at lib/fast_rcnn/train.py.
As I mentioned, it worked for loss. But the problem is that exporting feature maps by image_summary needs to reshape the tensor(in this case, the weights of rpn_bbox), whose shape(size) varies in the train.py. Does anyone know how to fix this? Or I have to do this in another way?
Sorry for such long questions.
